### DISCUSSION TITLE
Stories attachments don't support SVG uploads

### DISCUSSION THREAD

**Discussion by andersan** [Terrastories/terrastories#721] [Temporarily Archived Tickets]
Minor issue, not likely to matter for actual users. Don't know if other file types are also not supported.

Steps to reproduce:
1. edit a story
2. upload an SVG
3. press the update button
4. try to edit the story again, and see below message

Error message: 
![Screenshot from 2022-03-03 09-33-33](https://user-images.githubusercontent.com/12814577/156585753-48cfbc21-fb4a-4359-aecf-f0aa89723646.png)

Afterwards, I can't access the story to edit it at all.

**Comment by rudokemper** [comment]
Thanks @andersan. We're moving this to discussion for now but will revisit after some refactor work which will also hopefully solve this.
